packages/core/src/commands/snapshot-verify.ts

```diff
@@ -35,10 +41,10 @@ export class Command extends Commands.Command {
         this.definition
             .setFlag("token", "The name of the token.", Joi.string().default("ark"))
             .setFlag("network", "The name of the network.", Joi.string().valid(...Object.keys(Networks)))
-            .setFlag("skipCompression", "Skip gzip compression.", Joi.boolean())
-            .setFlag("trace", "Dumps generated queries and settings to console.", Joi.boolean())
-            .setFlag("blocks", "Blocks to verify, correlates to folder name.", Joi.string())
-            .setFlag("verifySignatures", "Verify signatures of specified snapshot.", Joi.boolean());
+            // .setFlag("skipCompression", "Skip gzip compression.", Joi.boolean())
```

Comment:
Is this now unused or will it be implemented?

Reply:
Unused and removed.